agent.go

```diff
@@ -385,6 +387,101 @@ func (c *controller) agentClose() {
 	agent.networkDB.Close()
 }
 
+// Task has the backend container details
+type Task struct {
+	Name       string
+	EndpointID string
+	EndpointIP string
+	Info       string
+}
+
+// ServiceInfo has service specific details along with the list of backend tasks
+type ServiceInfo struct {
+	VIP   string
+	Tasks []Task
+	Ports []string
```

Comment:
Agree. Its good to include the `FWMarker`. Instead of showing it as `FWMarker` which is very implementation specific we can call it `Local LB ID` which also conveys the id is local to the node and need not be same across all nodes.